### PR TITLE
EdmDecimal Precision = Scale fix

### PR DESCRIPTION
**Pull request by AlapanGhosh**
Decimal Precision can now be same as Scale.
EdmDecimal generator would generate values between 0 and 1.

### CODE REVIEW

**Comment by phanak-sap** [comment]
This seems to be fix for the issue #92. But I am puzzled, that part of the PR are updates of the test (direct builder basicaly) which has no relevant to the Edm.Decimal.. and on contrary, no tests for the updated generator method  that would ideally reproduced the bug and then pass with the fix.

File: odfuzz/generators.py [hunk 1]
```diff
@@ -88,10 +88,14 @@ def generate(generator_format='uri'):
 class EdmDecimal:
     @staticmethod
     def generate(self,generator_format='uri'):
-        divider = random.randint(1, 10 ** self.scale)
-        scale_range = random.randint(0, self.scale)
-        rand_int = random.randint(1, (10 ** (self.precision - scale_range)) - 1)
-        value = '{0:.{1}f}'.format(rand_int / divider, scale_range) + 'm'
+        if self.precision == self.scale:
```
Comment (phanak-sap) [inline odfuzz/generators.py:91]: I guess it is OK, but if there were all generator.py test passing with this bug, then there is one test case definitelly missing

**Review by phanak-sap — CHANGES_REQUESTED** [review]

**Comment by AlapanGhosh** [comment]
> 
> 
> This seems to be fix for the issue #92. But I am puzzled, that part of the PR are updates of the test (direct builder basicaly) which has no relevant to the Edm.Decimal.. and on contrary, no tests for the updated generator method that would ideally reproduced the bug and then pass with the fix.

Those are minor changes suggested in last PR which have been added in this one as discussed. Its not part of the fix. 
A new test for precision == scale will be added

File: odfuzz/generators.py (hunk 1 above)
Comment (AlapanGhosh) [reply odfuzz/generators.py:91]: addressed

**Review by phanak-sap — APPROVED** [review]

**Pull request merged**